person.rb

```diff
@@ -32,6 +34,10 @@ def of_age?
 
     false
   end
+
+  def add_book
+    @rentals << rental
+  end
```

Comment:
- [ ] Like the previous method of adding rentals, please create the new rental inside this method so you can use `self` to create a new rental for this person.

```ruby
def add_rental(book, date)
   Rental.new(date, book, self)
end
```

Reply:
Thank you.